crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -94,12 +94,13 @@ impl VM {
     }
 
     // CALLCODE operation
+    // TODO: Add gas consumption
```

Comment:
Create the issue for this, and link it here or add the issue's number in this comment

Reply:
🫡

Reply:
Done in [1ff2d77](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1029/commits/1ff2d77272f33be8879304c3e849b0b39c95df9d).